photon-lib/build.gradle

```diff
@@ -7,7 +7,7 @@ apply plugin: "edu.wpi.first.NativeUtils"
 import java.nio.file.Path
 
 ext {
-    nativeName = "photonlib"
```

Comment:
These really should be lowercase, this would also cause the vendordep to break which we want to strickly avoid. Is there a specific reason why this change resolves the issue?

Reply:
[def baseArtifactId = nativeName](https://github.com/PhotonVision/photonvision/blob/68d8a943f7795a68283a714b156d043aaaa00582/shared/javacommon.gradle#L11)

This line makes the artifact id based on the nativename. Thats why this fixes the maven dir issue

Reply:
Why would the vendor dep break?

Reply:
Because the vendordep is looking for that, previously it was Title case but now it is all lowercase, old vendor deps are looking for Title case, whereas all beta vendordeps are looking for lowercase. Going back would break all beta vendordeps.